apps/root/src/pages/position-detail/frame/index.tsx

```diff
@@ -117,460 +81,76 @@ const PositionDetailFrame = () => {
-    } catch (e) {
-      // User rejecting transaction
-      // eslint-disable-next-line no-void, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
-      if (shouldTrackError(e as Error)) {
-        trackEvent('DCA - Position details - Withdraw error', { chainId, useProtocolToken });
-        // eslint-disable-next-line no-void, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
-        void errorService.logError('Error while withdrawing', JSON.stringify(e), {
-          position: position.id,
-          useProtocolToken,
-          chainId: position.chainId,
-        });
-      }
-      /* eslint-disable  @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access */
-      setModalError({
-        content: <FormattedMessage description="modalErrorWithdraw" defaultMessage="Error while withdrawing" />,
-        error: {
-          code: e.code,
-          message: e.message,
-          data: e.data,
-          extraData: {
-            useProtocolToken,
-            chainId: position.chainId,
-          },
-        },
-      });
-      /* eslint-enable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access */
-    }
-  };
-
-  const onWithdrawFunds = async (useProtocolToken = true) => {
-    if (!position) {
-      return;
-    }
-    try {
-      const hasYield = position.from.underlyingTokens.length;
-      let hasPermission = true;
-      if (useProtocolToken || hasYield) {
-        hasPermission = await positionService.companionHasPermission(position, PERMISSIONS.REDUCE);
-      }
-      const protocolOrWrappedToken = useProtocolToken ? protocolToken.symbol : wrappedProtocolToken.symbol;
-      const fromSymbol =
-        position.from.address === PROTOCOL_TOKEN_ADDRESS || position.from.address === wrappedProtocolToken.address
-          ? protocolOrWrappedToken
-          : position.from.symbol;
-
-      const removedFunds = position.rate.amount * BigInt(position.remainingSwaps);
-      setModalLoading({
-        content: (
-          <>
-            <Typography variant="body">
-              <FormattedMessage
-                description="Withdrawing funds from"
-                defaultMessage="Withdrawing {fromSymbol} funds"
-                values={{ fromSymbol }}
-              />
-            </Typography>
-            {useProtocolToken && !hasPermission && hasSignSupport && (
-              <Typography variant="body">
-                <FormattedMessage
-                  description="Approve signature companion text"
-                  defaultMessage="You will need to first sign a message (which is costless) to authorize our Companion contract. Then, you will need to submit the transaction where you get your balance back as {from}."
-                  values={{ from: position.from.symbol }}
-                />
+          <StyledTab
+            disableRipple
+            sx={{ marginLeft: '32px' }}
```

Comment:
Gotcha! no px :D

Reply:
I took the time to apply styles to the whole frame based on this comment 😄 . We didn't have a task so now it's included here